### ISSUE TITLE
Accessibility – hyperlinks are invisible 

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#1994]
From kiwix-android created by [Paol-M](https://github.com/Paol-M): kiwix/kiwix-android#3482  

Hyperlinks are indistinguishable from other text for completely colorblind individuals or those who use grayscale mode on their phone.

**Comment by kelson42** [comment]
Or sleep mode really, making it even more important.
I'd suggest maybe underline, italicize, or bolden links.

**Comment by kelson42** [comment]
which zim file do we talk about exactly?

**Comment by kelson42** [comment]
The issue is in Wikipedia, Wiktionary, and probably many other ZIMs.

**Comment by kelson42** [comment]
@Paol-M But the link are exactly like on the online version!? So you have the same problem with en.wikipedia.org for example?

**Comment by kelson42** [comment]
No, I don't have the same problem. On the online version, links appear as a slightly different shade of black, and, in the setting, I have the option to enable underlining links. In the Kiwix app for Android, links and non-links are indistinguishable in both dark and light mode.

**Comment by kelson42** [comment]
Moving to `openzim/mwoffliner

**Comment by Jaifroid** [comment]
@kelson42 If we could revive the ZIM contents API idea, this would be an ideal candidate. It would be extremely easy to alter hyperlink appearance in CSS, but so as to avoid each reader implementing it in a custom way, it would be much better to have an API for style customization.

On the other hand, I suppose the issue might not be confined to mwOffliner.

**Comment by kelson42** [comment]
No example, not feedback from the reporter and no concrete example. In addition we have nowmfixed MWoffliner to really better handle the CSS... so pretty sure this has been fixed.

**Comment by aueibu** [comment]
Hi there, just got the Wikipedia Zim up and running for Kiwix Android on a Boox ereader. I'm having the same trouble as the original post, I cannot reliable see what parts of the article are hyperlinks or not. It was mentioned that this is now solved? How can I learn more about how to change this so that hyperlinks appear more visible for a monochrome display? Thanks!

**Comment by kelson42** [comment]
@auedpo I guess you have exactly the same problem with Wikipedia online? We don't change anything.
